subprojects/ce-login/celogin/include/CeLogin.h

```diff
@@ -240,26 +246,37 @@ struct AcfUserFields
  *  @param publicKeyLengthParm the length of the provided public key
  *  @param serialNumberParm a pointer to the serial number of the current system
  *  @param serialNumberLengthParm the length of the provided serial number
- *  @param expirationTimeParm a unix timestamp indicating when the ACF expires
+ *  @param currentReplayIdParm the current replay ID persisted by the BMC. The very first time
+ *                             this function is called, the BMC may pass zero as input.
+ *  @param updatedReplayIdParm the value the BMC is required to persist as the new replay ID if
+ *                             the function call succeeds. This must be passed in as the
+ *                             currentReplayIdParm the next time this interface is called.
+ *  @param acfTypeParm the AcfType parsed from the provided file (e.g. service or admin reset)
+ *  @param expirationTimeParm the time at which this ACF expires
  *
  *  @return A CeLoginRc indicating the result. CeLoginRc::Success indicates that the ACF is
- *          valid (excluding password validation).
+ *          valid and the AcfUserFields has been populated with the relevant fields from the
+ *          file. On success, the updatedReplayIdParm must be persisted and used on the next
+ *          call to this interface.
  */
-CeLoginRc checkAcfIntegrityV2(
+CeLoginRc verifyACFForBMCUploadV2(
```

Comment:
I'm confused... admin reset is only performed in the upload flow, how is the BMC going to retrieve the auth code?

Or is the BMC expected to call both functions in the upload flow?

Reply:
Ah, that is a problem. Will think on that.